src/pages/ShopingCart.page.js

```diff
@@ -7,10 +7,18 @@ export class ShopingCartPage extends BaseSwagLabPage {
 
     removeItemSelector = '[id^="remove"]';
 
+    checkoutBtnSelector = '[id^="checkout"]';
+
     get headerTitle() { return this.page.locator('.title'); }
 
     get cartItems() { return this.page.locator(this.cartItemSelector); }
 
+    inventoryItemName = '.inventory_item_name';
+
+    inventoryItemDescription = '.inventory_item_desc';
+
+    inventoryItemPrice = '.inventory_item_price';
+
```

Comment:
here as well, lets have a structure. In the 4-11 lines we have fields then we have methods and again fields